packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -801,6 +801,12 @@ export const Table = <
                             column.resizeWidth = el.getBoundingClientRect().width;
                           }
                         }}
+                        tabIndex={0}
```

Comment:
I think it should only be focusable if it can be sorted.

Reply:
I did it this way because it followed how the mouse clicks/hovers work for the react table. The header cells change color on hover, so I made it so that all header cells can be tab-focusable.

I changed it to be focusable only if sortable though.

Reply:
@adamhock What do you think?

Reply:
I agree with Vilius. I think that unless there's an action you can take on the header cells (such as sorting), then it should not be focusable.